CONTRIBUTING.md

```diff
@@ -0,0 +1,160 @@
+# Contributing
+
+First of all, read how to deploy a local copy of Pan Docs in the [README](https://github.com/gbdev/pandocs#contributing).
+
+## Git
+
+- Commit messages must be meaningful, avoid "Update XXX.md". Please describe what you changed, how, and **why**.
+- If it is desirable to squash a PR, but not into a single commit, keep in mind that most of the time, maintainers have push access to branches being PR'd ("Allow edits and access to secrets by maintainers" checkbox in the right sidebar).
+- Maintainers: rebase or squash PRs, please do not create merge commits. Avoid leaving formatting/typos/etc. commits in `master`'s history.
+
+## [Board](https://github.com/gbdev/pandocs/projects/1)
+
+Some notes on how we use the Project board:
+
+**Triage**: the issues here are still in the RFC phase. The discussion is on-going or not enough information is available to progress. You're welcome to join the discussion, provide references and propose solutions.
+
+**To Do**: stuff here is ready to be worked on. You should have all the information you have to work on a PR to implement the solution required by the issues.
+
+## Document Style
+
+### 1. Conventions documentation
+
+A short paragraph introducing and clarifying the conventions used in the document must be placed (and kept updated) in the top of the document.
+
+### 2. Monospaced text, code snippets, code boxes
+
+- No ligatures
+
+### 3. Pseudocode
+
+- Assignment: :=
+- Comparison: =, ≠, <, >, ≤, ≥
+- Arithmetic: +, −, ×, /, mod
+- Floor/ceiling: ⌊, ⌋, ⌈, ⌉.     a :=    ⌊b⌋    + ⌈c⌉
+- Logical: and, or, not
+- Sums, products: Σ Π
+
+References:
+1. http://www.cs.cornell.edu/courses/cs482/2003su/handouts/pseudocode.pdf
+2. https://blog.usejournal.com/how-to-write-pseudocode-a-beginners-guide-29956242698
+3. https://cs.wmich.edu/gupta/teaching/cs3310/sp18cs3310web/lecture%20notes%20cs3310/PseudocodeBasics.pdf
+4. https://en.wikipedia.org/wiki/Pseudocode#Mathematical_style_pseudocode
+
+### 4. Units and prefixes:
+
+Use Binary Prefixes (Ki, Mi) for Bytes. E.g.:
+
+```
+1 MiB = 2^20 bytes = 1,048,576 bytes
+1 KiB = 2^10 bytes = 1,024 bytes
+```
+
+- Abbreviate bytes as B when used with a prefix. Spell out bits as bits, even when used with a prefix to avoid ambiguity. When used as a unit suffix, unit names are always singular, so it should be "20 kbit", not "20 kbits". (Also, "bytes" as a word is fine, just like "30 meters" is a valid way of talking about that length. But "Mbyte" and the like should go away.)
```

Comment:
```suggestion
- Abbreviate bytes as "B" when used with a prefix. Spell out bits as "bits", even when used with a prefix, to avoid ambiguity. When used as a unit suffix, unit names are always singular, so it should be "20 kbit", not "20 kbits". (Also, "bytes" as a word is fine, just like "30 meters" is a valid way of talking about that length. But "Mbyte" and the like should go away.)
```